cmd/external-repos/main.go

```diff
@@ -48,33 +48,34 @@ func main() {
 		log.Debug().Msg("Successfully loaded external repositories.")
 	} else if args[1] == "introspect" {
 		if len(args) < 3 {
-			log.Error().Msg("Usage:  ./external_repos introspect URL [--force]")
+			log.Error().Msg("Usage:  ./external_repos introspect [--force] URL [URL2]...")
 			os.Exit(1)
 		}
-		url := args[2]
-		if len(args) > 3 {
-			flagset := flag.NewFlagSet("introspect", flag.ExitOnError)
-			flagset.BoolVar(&forceIntrospect, "force", false, "Force introspection even if not needed")
-			if err = flagset.Parse(args[3:]); err != nil {
-				log.Error().Err(err).Msg("Usage:  ./external_repos introspect URL [--force]")
-				os.Exit(1)
```

Comment:
note that we should probably switch to a better arg parsing library overall, but i didn't want to do it as part of this change